app.js

```diff
@@ -0,0 +1,185 @@
+var timeLabelsArray = ['10AM: ','11AM: ','12PM: ', '1PM: ', '2PM: ', '3PM: ', '4PM: ', '5PM: ', 'SUM: ']
+var pikePlace = {
+  min: 17,
+  max: 88,
+  avg: 5.2,
+  randomOutput: function(){
+    return Math.floor(((Math.random()*(this.max - this.min + 1)) + this.min) * this.avg);
+  },
+  dailyRandomOutput: function(){
+    var cookiesArray = [];
+    for(i = 0; i < 8; i++){
+      var arrayValue = this.randomOutput();
+      cookiesArray.push(arrayValue);
+    }
+    return cookiesArray;
+  },
+  randomOutputSum: function(){
+    var arraySum = this.dailyRandomOutput();
+    var sum = 0;
+    for(i = 0; i < arraySum.length; i++){
+      sum += arraySum[i];
+    }
+    arraySum.push(sum);
+    console.log(arraySum);
+    return arraySum;
+  },
+  renderManipulation: function(){
+    var arrSum = this.randomOutputSum();
+    var x = document.getElementById('pike_place');
+    for(i=0;i<9;i++){
```

Comment:
Your other for loops aren't squished with the numbers and operators the way this one is. Any reason for doing this one differently?